src/cachet_endpoint.txt

```diff
@@ -0,0 +1 @@
+https://demo.cachethq.io/
```

Comment:
Perhaps # update-me-first or an invalid URL or something that doesn't cause traffic to hit the nice people at cachethq's demo site.